session4/Assignment1/Assignment1.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.List;
+
+class Assignment1
+{	
+	List<String> list = new ArrayList<String>();	//creating Array list of string

```

Comment:
Why ArrayList is assigned into List rather than ArrayList.?

Reply:
both are possible. so i took list.